nzshm_common/location/location.py

```diff
@@ -63,7 +63,7 @@
 }
 
 
-def _lat_lon(_id) -> Optional[LatLon]:
+def lat_lon_by_id(_id) -> Optional[LatLon]:
```

Comment:
@chrisdicaprio I think a docstring is needed here as we're adding to the API. Can you please check it's being picked up in mkdocs also?